spaceapi/src/utils.rs

```diff
@@ -3,7 +3,7 @@
 /// An ``Optional`` can contain ``Optional::Value<T>`` or ``Optional::Absent``.
 /// It is similar to an ``Option``, but ``Optional::Absent`` means it will be
 /// omitted when serialized, while ``None`` will be serialized to ``null``.
-#[derive(Debug, Copy, Clone)]
+#[derive(Clone, Copy, PartialEq, PartialOrd, Eq, Ord, Debug, Hash)]
```

Comment:
OK, I think `Absent == Absent` can make sense (quite in contrast to `NaN == NaN` or `Infinity == Infinity`).